examples/learning/learn_associations.ipynb

```diff
@@ -0,0 +1,330 @@
+{
+ "metadata": {
+  "kernelspec": {
```

Comment:
What version of IPython did you use to create this notebook? Our [existing notebooks](https://raw.githubusercontent.com/nengo/nengo/master/examples/basic/2d_representation.ipynb) don't have this metadata in them. Ideally, all of the notebooks would be created with the same version of IPython, which I think for the others is `IPython<3` (2.4.1 now I think).